include/LLGL/Backend/OpenGL/Linux/LinuxNativeHandle.h

```diff
@@ -26,7 +35,19 @@ namespace OpenGL
 */
 struct RenderSystemNativeHandle
 {
-    GLXContext context;
+    union
+    {
+        LLGL_DEPRECATED("Deprecated since 0.04b; Use glx instead.")
+        GLXContext context;
+        GLXContext glx;
+        #if LLGL_EXPOSE_WAYLAND || LLGL_LINUX_ENABLE_WAYLAND
+        EGLContext egl;
+        #else
+        void*      egl;
+        #endif
+    };
+
+    RenderSystemNativeHandleType type;
```

Comment:
Can you make `type` the first field in this struct? `Direct3D12::ResourceNativeHandle` does the same. So the first bytes read are always identifying how to interpret the rest of the union.